src/components/Executions/filters/types.ts

```diff
@@ -20,13 +20,15 @@ export interface FilterButtonState {
     onClick: () => void;
 }
 
-export type FilterStateType = 'single' | 'multi' | 'search';
+export type FilterStateType = 'single' | 'multi' | 'search' | 'boolean';
 
 export interface FilterState {
     active: boolean;
     button: FilterButtonState;
+    hidden?: boolean;
     label: string;
     type: FilterStateType;
+    status?: string;
```

Comment:
This could be removed if you implement my suggestion above regarding querying the user profile directly in `EntityExecutions`